### PR TITLE
Metadata without assets file

### PR DESCRIPTION
**Pull request by pepoviola**
Hi @dgbeck,
                      When you have time can you review this, it's the first approach in the way to support the `cdn` feature.

Thx!

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -805,14 +807,16 @@ Cartero.prototype.writeMetaDataFile = function( callback ) {
 	}, {} );
 
 	var metaData = JSON.stringify( {
-		formatVersion : 3,
+		formatVersion : 4,
 		packageMap : packageMap,
 		entryPointMap : entryPointMap,
+		entryPoints : _this.entryPoints,
```
Comment (dgbeck) [inline index.js]: Hi @pepoviola we going to call this something else... no? Like a big long name, I can't remember what tho...

Comment (pepoviola) [inline index.js]: Hi @dgbeck, I double check and this should be call `assetsRequiredByEntryPoint` with this structure:
```javascript
assetsRequiredByEntryPoint : {
  'view/blah/blah.es' : {
    script: [ ],
    style: [ ],
    image: [ ]
  }
}
```

I would change this.

Thx!

**Comment by pepoviola** [comment]
Hi @dgbeck renamed here and in the `cartero-node-hook`.

Thx!

**Comment by dgbeck** [comment]
Cool thx!

**Pull request merged**